src/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import { get } from './utils/api';
+import './index.css';
+
+function ArticleCard() {
+    const [articles, setArticles] = useState([]);
```

Comment:
Could we also get some loading state to give the user feedback whilst the API is fetching the data?